app/static/src/app/types/languageTypes.ts

```diff
@@ -0,0 +1 @@
+export type Language = "en" | "fr"
```

Comment:
Probably daft question but why not an enum here?

Reply:
saves us having to write Language.en everywhere basically, other than that, no real difference

Reply:
changed to enum now and replaced all instances of "en" and "fr"